components/feedstock/stores.js

```diff
@@ -88,38 +85,47 @@ const Store = ({ dataset }) => {
 
           <Button
             as={Link}
-            href={`https://data-viewer-git-katamartin-pyramid-maps-carbonplan.vercel.app/?dataset=${pyramid || url}`}
+            href={`https://data-viewer-git-katamartin-pyramid-maps-carbonplan.vercel.app/?dataset=${
+              pyramid || url
+            }`}
             target='_blank'
             rel='noopener noreferrer'
-            suffix={<RotatingArrow sx={{ ml: '2px' }} />}
+            suffix={<RotatingArrow />}
             sx={{
-              mt: 3,
-              mb: 2,
-              fontSize: 0,
+              textTransform: 'uppercase',
               fontFamily: 'mono',
               color: 'secondary',
-              textTransform: 'uppercase',
+              mt: 3,
```

Comment:
If you take a look at the [`Button` component](https://github.com/carbonplan/components/blob/main/src/button.js#L34-L38) in `@carbonplan/components`, the styles of the button text and the `suffix` element are customized to be dialed for each specific button `size` across different breakpoints. All of this dialing is very specific to the default fontFamily for the `Button` component.

This basically means that if you want to deviate from the standard fontFamily (as you are by using `mono`) and/or the standard fontSizes (as it sounds like you're planning to), the same styles that are dialed for each `size` in the `Button` component are the ones you'll want to adjust here. I think the main style parameters you might adjust are listed [here](https://github.com/carbonplan/components/blob/main/src/button.js#L34-L38) and [here](https://github.com/carbonplan/components/blob/main/src/utils/get-size-styles.js#L8-L13). Also because of the responsive styles defined in `Button`, it's important to test that your adjustments work across breakpoints!

Also if you're going to drop the fontSize further, I would start by dropping the `size` prop.

Reply:
Looking at your latest deployment it seems like you'll likely want to tweak the positioning of the `suffix` element (the `RotatingArrow`).

While you can't directly override the [transformation that `Button` establishes](https://github.com/carbonplan/components/blob/main/src/button.js#L38), I think you can play with adding a `transform` to the `RotatingArrow` component to counteract the `Button` style. The icon is too low, so I think you'd want a negative `translateY` transformation.

<img width="190" alt="CleanShot 2024-04-18 at 10 16 46@2x" src="https://github.com/carbonplan/leap-data-catalog/assets/12436887/9ebb6482-a8a6-4c76-98a2-189085b4c39e">